client/src/components/timetable/CompreSched.jsx

```diff
@@ -27,4 +29,10 @@ const CompreSched = props => {
   );
 };
 
-export default CompreSched;
+const mapStateToProps = state => {
+  return {
+    myCourses: state.updateTT.myCourses
+  };
+};
```

Comment:
Is `myCourses` being changed here? If not, do we need the `mapStateToProps` here?

Reply:
nope.ofc we do need it .This is one of many ways of how we access state in redux!